### DISCUSSION TITLE
Is it possible to use multiple vouchers for one transaction?

### DISCUSSION THREAD

**Discussion by steven-morgan** [pretix/pretix#4671] [Q&A] [answered]
I can't see a way to use multiple vouchers for one transaction, is this possible? If I have a customer who receives a voucher from being on the waiting list, but they also have a different independent discount voucher, is it possible to use these two together to apply the discount to the waiting list ticket?

**Comment by raphaelm** [comment, marked as answer]
If one "transaction" means the same ticket, as in your example, then: no.
